processor/k8sattributesprocessor/processor_test.go

```diff
@@ -727,6 +727,78 @@ func TestProcessorAddLabels(t *testing.T) {
 	}
 }
 
+func TestAddNamespaceLabels(t *testing.T) {
```

Comment:
Should we also add a test `TestAddNodeLabels`?

Reply:
Sure, added